<?xml version="1.0" encoding="UTF-8"?><testrun name="Rerun uk.co.luciditysoftware.campervibe.tests.domain.entities.booking.ValidateMakeTests.InvalidRequestFails" project="Campervibe" tests="1" started="1" failures="0" errors="1" ignored="0">
  <testcase name="InvalidRequestFails" classname="uk.co.luciditysoftware.campervibe.tests.domain.entities.booking.ValidateMakeTests" time="414.748">
    <error>java.lang.NullPointerException&#13;
	at uk.co.luciditysoftware.campervibe.tests.domain.entities.booking.ValidateMakeTests.lambda$1(ValidateMakeTests.java:91)&#13;
	at java.util.stream.MatchOps$1MatchSink.accept(Unknown Source)&#13;
	at java.util.ArrayList$ArrayListSpliterator.tryAdvance(Unknown Source)&#13;
	at java.util.stream.ReferencePipeline.forEachWithCancel(Unknown Source)&#13;
	at java.util.stream.AbstractPipeline.copyIntoWithCancel(Unknown Source)&#13;
	at java.util.stream.AbstractPipeline.copyInto(Unknown Source)&#13;
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)&#13;
	at java.util.stream.MatchOps$MatchOp.evaluateSequential(Unknown Source)&#13;
	at java.util.stream.MatchOps$MatchOp.evaluateSequential(Unknown Source)&#13;
	at java.util.stream.AbstractPipeline.evaluate(Unknown Source)&#13;
	at java.util.stream.ReferencePipeline.anyMatch(Unknown Source)&#13;
	at uk.co.luciditysoftware.campervibe.tests.domain.entities.booking.ValidateMakeTests.InvalidRequestFails(ValidateMakeTests.java:91)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#13;
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)&#13;
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</error>
  </testcase>
</testrun>
